requirements.txt

```diff
@@ -1,4 +1,9 @@
-Django==2.2.28
-djangorestframework==3.11.2
-mysqlclient==1.3.9
-pytz==2019.3
+# Pin packages to support and work with py3.6.
+Django==3.1.14
+djangorestframework==3.15.1
+mysqlclient==2.1.1
+pytz==2024.2
+
+# Below are the additional packages
```

Comment:
How are they "additional"? If they're no different to the other packages. Just include them in the list above.

Reply:
I have added that because they are new ones that are getting added the base or previous ones. That's why I have added additional packages.  As now they are also required it make sense to merge them, Adrian. Updated in ab8cc8390cdc5bd805f82f8b93e7272cb430de80